openstack/rds/v3/configurations/results.go

```diff
@@ -60,10 +90,14 @@ func (r CreateResult) ExtractInto(v interface{}) error {
 	return r.Result.ExtractIntoStructPtr(v, "configuration")
 }
 
+type commonResult struct {
```

Comment:
There is no real sense to use `commonResult` ― there are no methods bind to the type.